adidas-ascii-logo/go/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+  "fmt"
+)
+func createAdidasAsciiLogo(width int) string {
+  return fmt.Sprintf("%d",width)
+}
+
+func main(){
+  fmt.Println(fmt.Sprintf("\nadidas (width %d)",2))
```

Comment:
If you shows only one example, use at least width `9`.

Reply:
Done